Project-II-Calculator/calculator/src/components/DisplayComponents/Display.css

```diff
@@ -0,0 +1,6 @@
+.displayPanel {
+    background-color: black;
+    margin: 0;
+    width: 300px;
```

Comment:
Don't forget to set the color of the display text and align it as shown in the design file. You can set the following styles to do this:
```
display: flex;
justify-content: flex-end;
align-items: center;
padding-right: 10px;;
```

_Note: For `padding-right` to work properly, the following selector must also have been added (be it in this file or another):_
```
* {
  box-sizing: border-box;
}
```